learn/cookbooks/vercel.mdx

```diff
@@ -0,0 +1,94 @@
+# Integrate Meilisearch Cloud with Vercel
+
+Getting a Meilisearch Cloud instance created and linked to your Vercel projects has never been easier. 
+
+## Introducing our tools
+
+### What is Vercel?
+
+[Vercel](https://vercel.com/) is a cloud platform for building and deploying web applications. It works out of the box with most popular web development tools.
+
+### What is Meilisearch Cloud?
+
+[Meilisearch Cloud](https://www.meilisearch.com/cloud) is a cloud-based SaaS platform, built on top of the Meilisearch open-source search engine.
+
+## Integrate Meilisearch into your Vercel project
+
+### Create and deploy a Vercel project
+
+From your Vercel dashboard, create a new project. You can create a project from a template, or import a Git repository.
+
+![Create a new project on Vercel dashboard](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vercel/01.create-new-project-on-vercel-dashboard.png)
+
+Select your project, then click on **Deploy**. Once deployment is complete, go back to your project’s dashboard. 
+
+### Add the Meilisearch Cloud integration
+
+Go to the project settings tab and click on **Integrations** on the sidebar menu to the left of your screen.
+
+![Selecting the integration tab in the project settings](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/vercel/02.project-settings-integrations-tab.png)
+
+Search for the [Meilisearch Cloud integration](https://vercel.com/integrations/meilisearch-cloud) in the search bar. Click on the **Add integration** button.
+
+<Capsule intent="warning">
+Meilisearch Cloud integration is currently in the beta phase. It's not accessible via Vercel's marketplace search at this time.  However, you can conveniently access the integration by visiting the dedicated [Meilisearch Cloud integration page](https://vercel.com/integrations/meilisearch-cloud) directly.
```

Comment:
I would make it shorter:

> Meilisearch integration is currently not accessible via Vercel's marketplace search. You can access the integration directly by visiting the [Meilisearch integration page](https://vercel.com/integrations/meilisearch-cloud).

if you want to write "meilisearch cloud integration" you can go for it, but i feel like "meilisearch integration" is more impactful